packages/test-helpers/src/test-injector.ts

```diff
@@ -1,16 +1,16 @@
 import { StrykerOptions } from '@stryker-mutator/api/core';
 import { Logger } from '@stryker-mutator/api/logging';
 import { commonTokens, PluginContext, PluginResolver } from '@stryker-mutator/api/plugin';
-import * as sinon from 'sinon';
+import sinon from 'sinon';
 import { Injector, createInjector } from 'typed-inject';
 
 import * as factory from './factory';
 
 class TestInjector {
-  public pluginResolver: sinon.SinonStubbedInstance<PluginResolver>;
-  public options: StrykerOptions;
-  public logger: sinon.SinonStubbedInstance<Logger>;
-  public injector: Injector<PluginContext>;
+  public pluginResolver!: sinon.SinonStubbedInstance<PluginResolver>;
```

Comment:
!? are they possibly undefined?

Reply:
Yes, this is because of `--strictPropertyInitialization` again. But this time I used a `!` (not null assertion) instead of a `?`, because I know for sure this can only _theoretically_ be undefined and will always be defined in practice because I call `reset` directly inside this file (and it's a singleton).